packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -843,6 +843,12 @@ class MediaQuery extends InheritedWidget {
     return MediaQuery.of(context, nullOk: true)?.platformBrightness ?? Brightness.light;
   }
 
+  /// Returns highContrast for the nearest MediaQuery ancestor or false, if no
+  /// such ancestor exists.
```

Comment:
This API doc should really incorporate a `[MediaQueryData.highContrast]` link somewhere.

Reply:
Done